### PR TITLE
Add WarfareMC.eu

### PR DESCRIPTION
**Pull request by Mirayii**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/warfaremc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "server_wildcards": [
+    "play.warfaremc.eu"
+  ],
+  "supported_languages": [
+    "cz",
+    "sk"
+  ],
+  "social": {
+    "web": "https://warfaremc.eu",
+    "web_shop": "https://shop.warfaremc.eu",
```
Comment (JakobBruening) [inline minecraft_servers/warfaremc/manifest.json:14]: https://store.warfaremc.eu/ would be the direct link ig.

File: minecraft_servers/warfaremc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "twitter": "warfaremceu",
+    "discord": "https://discord.gg/C3Vn4aY",
+    "tiktok": "warfaremc",
+	"instagram": "warfaremceu",
```
Comment (JakobBruening) [inline minecraft_servers/warfaremc/manifest.json:18]: fix formatting

File: minecraft_servers/warfaremc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      "versions": "1.16.5<*"
+    },
+    "smp": {
+      "name": "Public SMP",
+      "versions": "1.16.5<*"
+    }
+  },
+  "discord": {
+    "server_id": 252826895542910976,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/warfaremc/manifest.json:41]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/warfaremc/manifest.json (hunk 1 above)
Comment (Mirayii) [reply minecraft_servers/warfaremc/manifest.json]: Fixed url, should be fine now.

File: minecraft_servers/warfaremc/manifest.json (hunk 2 above)
Comment (Mirayii) [reply minecraft_servers/warfaremc/manifest.json]: Should be fixed, idk how i really tabbed there :S

File: minecraft_servers/warfaremc/manifest.json (hunk 3 above)
Comment (Mirayii) [reply minecraft_servers/warfaremc/manifest.json]: Changed.

**Pull request merged**